src/Tizen.NUI/src/public/Application/NUIApplication.cs

```diff
@@ -157,6 +157,22 @@ public NUIApplication(Size2D windowSize, Position2D windowPosition, ThemeOptions
             ApplyThemeOption(option);
         }
 
+        /// <summary>
+        /// The constructor with a stylesheet, window mode, window size, position and default window type.
+        /// </summary>
+        /// <param name="styleSheet">The styleSheet URL.</param>
+        /// <param name="windowMode">The windowMode.</param>
+        /// <param name="windowSize">The window size.</param>
+        /// <param name="windowPosition">The window position.</param>
+        /// <param name="type">The default window type.</param>
+        /// <since_tizen> 9 </since_tizen>
+        /// This should be hidden. Because the major purpose is to support IME window as keyboard application.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public NUIApplication(string styleSheet, WindowMode windowMode, Size2D windowSize, Position2D windowPosition, WindowType type) : base(new NUICoreBackend(styleSheet, windowMode, windowSize, windowPosition, type))
```

Comment:
is `stylesheet` the URL of a resource for background image of IME? I couldn't get what `stylesheet` is, so I am just asking.

Reply:
It is not sure how to use by IME. It just supports the service all of them.
After feedback by Input team, I will change this API's interface.